README.md

```diff
@@ -4,22 +4,56 @@ Klesk is a Slack service to store and return ladder data for Quake Live or whate
 [![Build Status](https://travis-ci.org/TheOneYouDontLike/Klesk.svg?branch=master)](https://travis-ci.org/TheOneYouDontLike/Klesk)
 
 ## Features
-- Creating new ladder
-- Joining existing ladder
-- Adding match result
-- Showing user statistics for ladder
-- Showing ranking
+- `newladder <laddername> [<keyword>]`<br/>
```

Comment:
it looks better - without them markdown doesn't give you line break, but if you put two linebreaks then there's an empty line between lines of text :d